contracts/PoolManager.sol

```diff
@@ -144,11 +152,11 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC1155, IERC1155Rec
     }
 
     /// @inheritdoc IPoolManager
-    function lock(bytes calldata data) external override returns (bytes memory result) {
+    function lock(bytes calldata hookData) external override returns (bytes memory result) {
```

Comment:
should this be called hookData? the data that gets passed into lock does not necessarily have to be for hooks

Reply:
bahh no!! thanks. lol so many typos. everything's called data, it was hard not to grab some extra ones in there